tests/test-helper.js

```diff
@@ -3,10 +3,6 @@ import {
   setResolver
 } from 'ember-qunit';
 import { start } from 'ember-cli-qunit';
-import { settings } from 'ember-native-dom-helpers';
-import config from '../config/environment';
```

Comment:
@simonihmig I think we want to keep this, the thing about the root element is that it can be configurable, originally we set up like so: https://github.com/cibernox/ember-native-dom-helpers/blob/master/tests/dummy/config/environment.js#L43

So can just change the line in the dummy app config to agree with the other change `rootElement: '#ember-testing > .ember-view'`

Reply:
Yeah, I noticed how this was working before, but afaict the changes here only affect the internal tests in this addon. The root element should still be configurable I think, as `tests/test-helper.js` does not affect the consumer of this addon, it's just internal, right?

And if we override the setting in the dummy config, we cannot really make sure in our tests that the default setting is what we actually expect. E.g. one could change something in `settings.js` that breaks the behavior, but the tests would not detect this.

Reply:
@simonihmig This code was here as an example of how to deal with setting a custom root element, preferring the config value over something hardcoded in this addon. If the is now another way to do that now then this example can be removed.

Reply:
> If the is now another way to do that now then this example can be removed.

No, there is no other way. It works just the same as before, this PR does not touch this!

The example in the Readme is still valid: https://github.com/cibernox/ember-native-dom-helpers#standard-dom-elements-returned-using-a-findfindall-helpers

As I said in the comment above, I removed this from `tests/test-helper.js` because it affects the addon's tests (and only those!). If this was left in place, it would obfuscate the default setting for the test I added [here](https://github.com/cibernox/ember-native-dom-helpers/pull/121/files#diff-c44eb6de9b796fe0f3758d69c2214971). The default setting could be broken, but the addon's tests would still all be passing, because the selector would be overridden in `tests/test-helper.js`. That's why I removed it, to really test with the default selector that any consuming app would see.

I just tested this PR with ember-bootstrap (where I had this issue): 
* overriding the rootElement [here](https://github.com/kaliber5/ember-bootstrap/blob/master/tests/test-helper.js#L16) is not needed anymore (i.e. `:first-child` works correctly with the default setting)
* overriding the rootElement this way still works